src/conduct.md

```diff
@@ -32,42 +32,75 @@ If you see someone who is making an extra effort to ensure our community is welc
+
+## 7. Reporting Unacceptable Behavior
 
-## 6. If You Witness or Are Subject to Unacceptable Behavior
+If you are subject to or witness unacceptable behavior, or have any other concerns, please notify a community organizer or volunteer as soon as possible. 
 
-If you are subject to or witness unacceptable behavior, or have any other concerns, please notify a community organizer or volunteer as soon as possible. At in-person events, organizers/volunteers can be identified by a "Volunteer" t-shirt. Outside of events, organizers can be reached via email at [team@vtcodecamp.org](mailto:team@vtcodecamp.org); **however, this email may not be monitored during in-person events, so it is important to find an organizer/volunteer at the event.**
+At in-person events, organizers/volunteers can be identified by a "Volunteer" t-shirt. Outside of events, organizers can be reached via email at [team@vtcodecamp.org](mailto:team@vtcodecamp.org); **however, this email may not be monitored during in-person events, so it is important to find an organizer/volunteer at the event or call the proper authorities based on the severity of the offense.**
 
 Additionally, community organizers are available to help community members engage with local law enforcement or to otherwise help those experiencing unacceptable behavior feel safe. In the context of in-person events, organizers will also provide escorts as desired by the person experiencing distress.
 
-**If you feel that you are unsafe in an emergency situation, contact 911.** For non-emergencies, the Burlington Police Department can be reached at (802) 658-2704, and the Champlain College office of Campus Public Safety can be reached at (802) 865-6465 (dial extension 6465 from any campus phone).
+**If you feel that you are unsafe in an emergency situation, contact 911**. For non-emergencies, the Burlington Police Department can be reached at (802) 658-2704, and the Champlain College office of Campus Public Safety can be reached at (802) 865-6465 (dial extension 6465 from any campus phone).
 
 If you are a witness to behavior that violates this Code of Conduct, we recommend you take the following actions:
 
-* If there is a threat present, contact 911, or the Burlington Police Department at (802) 658-2704. Additionally, the Champlain College office of Campus Public Safety can be reached at (802) 865-6465 (dial extension 6465 from any campus phone).
-* Sit with the victim and ask, "How can I help?"
-* Provide the victim with emergency contact information if needed.
-* Communicate with the event organizers about what happened and how to handle it.
+If there is an immediate threat present, contact 911, or the Burlington Police Department at (802) 658-2704. Additionally, the Champlain College office of Campus Public Safety can be reached at (802) 865-6465 (dial extension 6465 from any campus phone).
```

Comment:
I liked having these as bullet points because they seem more like actions to take. It seems a less clear call to action in paragraph form.

Reply:
Oh, I don't really know why I deleted that part. Adding back in. Thanks!